Cargo.toml

```diff
@@ -60,6 +60,7 @@ uhyve-interface = { version = "0.1.1", path = "uhyve-interface", features = ["st
 virtio-bindings = { version = "0.2", features = ["virtio-v4_14_0"] }
 rftrace = { version = "0.1", optional = true }
 rftrace-frontend = { version = "0.1", optional = true }
+sysinfo = "0.30.12"
```

Comment:
Here we can use `default_features=false`

Reply:
Addressed in https://github.com/hermit-os/uhyve/pull/701/commits/861a084cc1a5be6e9eb38054973b6fea8c497f5d, this doesn't remove the `windows` dependency problem - but I'll just leave it as-is for now as discussed.

I am still not totally sure why any dependency from the `[target.'cfg(target_os = "windows")'.dependencies]` would be included.